### PR TITLE
change Prefect server capitalization

### PR DESCRIPTION
**Pull request by seanpwlms**
### Checklist
- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: docs/host/index.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ tags:
     - SQLite 
 ---
 
-# Hosting Prefect Server
+# Hosting a Prefect Server
```
Comment (discdiver) [inline docs/host/index.md:15]:
```suggestion
# Hosting a Prefect server
```

File: docs/host/index.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ $ prefect config set PREFECT_API_URL="http://127.0.0.1:4200/api"
 
 
 !!! tip "`PREFECT_API_URL` required when running Prefect inside a container"
-    You must set the API Server address to use Prefect within a container, such a a Docker container. 
+    You must set the API server address to use Prefect within a container, such a a Docker container.
```
Comment (discdiver) [inline docs/host/index.md:66]:
```suggestion
    You must set the API server address to use Prefect within a container, such as a Docker container. 
```

**Review by discdiver — CHANGES_REQUESTED** [review]
Thank you! Found a few other items.

**Comment by seanpwlms** [comment]
@discdiver Good catches!

**Review by discdiver — APPROVED** [review]

**Review by billpalombi — APPROVED** [review]
Thank you!

**Pull request merged**